index.js

```diff
@@ -2,37 +2,77 @@
 
 const assert = require("assert");
 const NodeFetch = require("node-fetch");
-const supertest = require("supertest");
+const http = require("http");
 const url = require("url");
 const BrowserTab = require("./lib/BrowserTab");
 const {version} = require("./package.json");
-const {CookieAccessInfo, CookieJar} = require("cookiejar");
+const {CookieAccessInfo, CookieJar, Cookie} = require("cookiejar");
 const {normalizeHeaders, getLocationHost} = require("./lib/getHeaders");
-const saveToJar = require("./lib/saveToJar");
 
 module.exports = Tallahassee;
 
-const responseSymbol = Symbol.for("response");
+const kResponse = Symbol.for("response");
+const kOrigin = Symbol.for("origin");
 
 class OriginResponse {
-  constructor(uri, response, originHost, protocol) {
-    this[responseSymbol] = response;
-    const status = this.status = response.statusCode;
-    this.ok = status >= 200 && status < 300;
-    this.headers = new Map(Object.entries(response.headers));
-    this.url = originHost ? `${protocol}//${originHost}${uri}` : response.request.url;
+  constructor(response, originHost, protocol) {
```

Comment:
all responses are now from node fetch. we're basically just overriding the url-property, might be a good use case for Proxy / Reflect.